src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
```

Comment:
Let's change variable name to something more descriptive. (for example: `invertedRobot`)